### PR TITLE
added docker-compose configuration

### PR DESCRIPTION
**Pull request by eriksape**
This PR is for adding the setup for docker-compose

solves #160

### CODE REVIEW

File: development.dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+WORKDIR /usr/src/application/
+RUN apt-get install libevent-dev
+COPY . .
+RUN pip install -r requirements.txt
+
+CMD ["/bin/bash", "-c", "flask deploy-script && python main.py"]
```
Comment (TooAngel) [inline development.dockerfile:8]: Actually not sure what this is doing. I thought this would throw an error. Mind explaining what it does?

**Review by TooAngel — APPROVED** [review]
Nice, thanks a lot, looks good.

File: development.dockerfile (hunk 1 above)
Comment (eriksape) [reply development.dockerfile:8]: sorry you are right, was from a flask template i had and forgot about, i going to fix it

**Review by TooAngel — APPROVED** [review]
It would be cool and it speeds up the merge process when you squash your commits.

**Pull request closed without merging**